apluslms_roman/utils/path_mapping.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import PurePosixPath
+import json
+
+
+def get_host_path(original, mapping):
+    ret = original
+    orig_path = PurePosixPath(original)
+    for k, v in mapping.items():
+        try:
+            print(k, v)
+            relative_path = orig_path.relative_to(k)
+            ret = PurePosixPath(v).joinpath(relative_path)
+            return str(ret)
+        except ValueError:
+            print("Error when composing new path!")
+            return -1
```

Comment:
I presume the for loop should continue, so maybe `pass` or `continue´.

Reply:
I was thinking to return -1 so that builder will know there is error in composing from environment variable, so it will trie to using command line parameter or default instead.
But I forget that we could have multiple pair, in this case we also need to try pairs behind the failed one. so using pass is better